apps/inbound-mail/src/server/index.ts

```diff
@@ -0,0 +1,616 @@
+import { MailParser } from 'mailparser';
+import * as _ from 'lodash';
+import * as Promise from 'bluebird';
+import * as htmlToText from 'html-to-text';
+import * as events from 'events';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as request from 'superagent';
+import * as shell from 'shelljs';
+import * as util from 'util';
+import { SMTPServer } from 'smtp-server';
+import * as uuid from 'node-uuid';
+import * as dns from 'dns';
+import logger from './logger';
+import * as extend from 'extend';
+import { QueueService } from './queue-service';
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+const LanguageDetect = require('languagedetect');
+const mailUtilities = Promise.promisifyAll(require('./mailUtilities'));
+const queueService = new QueueService();
+
+class Mailin extends events.EventEmitter {
+  public configuration: IConfiguration;
+
+  private _smtp: any;
+
+  constructor() {
```

Comment:
As a small note, let's make sure we are not processing attachments for the time being and skip them here.